src/components/EditGrower.js

```diff
@@ -149,12 +162,7 @@ const EditGrower = (props) => {
             </Grid>
           ))}
 
-          <TextField
-            className={classes.textInput}
-            label="Grower-entered organization"
-            value={getValue('organization')}
-            disabled
-          />
+          <Box className={classes.textInput}>Organization - FCC [example]</Box>
```

Comment:
This isn't meant to be an example caption, it should display the value of `getValue('organization')` which is the text entered by the Treetracker app user when prompted for their organization.

The Admin Panel user's job is to match this free-text value to an organization from the dropdown list (`<SelectOrg>`). For example, the app user might enter "FCC ABC" and the Admin Panel user matches it to "FCCABC" in the list.

Reply:
"Thank you for your feedback. I understand that the caption should display the value of getValue('organization'), which is the text entered by the Treetracker app user when prompted for their organization. I will make the necessary changes and update the pull request shortly.